lib/express_validator.js

```diff
@@ -265,9 +265,18 @@ function validateSchema(schema, req, loc, options) {
     var paramErrorMessage = schema[param].errorMessage;
     delete schema[param].errorMessage;
 
+    var opts;
+
     for (var methodName in schema[param]) {
       validator.failMsg = schema[param][methodName].errorMessage || paramErrorMessage || 'Invalid param';
-      validator[methodName].apply(validator, schema[param][methodName].options);
+
+      opts = schema[param][methodName].options;
+
+      if (opts && !Array.isArray(opts)) {
```

Comment:
Let's be more explicit here and check for something like `opts != null` instead of just implicitly checking `opts`.